### ISSUE TITLE
Access to public groups/topics if logging in with "wrong" email

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#102]
Just sharing mine and @ilmartyrk's Slack conversation re [this help text](https://citizenos.com/help/login/troubleshooting-no-permission-to-access) as something to potentially consider. To me it seemed logical that they would be able to log in with any email address, as everyone can access the topic/group - there is no need for "permission". But maybe needs some thinking through re the pros and cons of doing it each way.

![image](https://user-images.githubusercontent.com/61320526/228482061-c29f4c08-f1a5-4b95-affd-4d5a4dfd01f9.png)

**Comment by BeccaMelhuish** [comment]
I feel like if we could make the process work so that the user is able to log in with whatever email address they wish, but is made aware of the fact that they are logging in with a different email address to the one they were invited with 
(if this is the case), and what the repercussions (?) would be of doing so, this could be preferable? 

But I'm not grasping the reason for why we want people to use the email address they were invited with (i.e. why we are still pushing them to do so even though access is open), or how serious they are. Might be something I'm missing! Is there some sort of scenario/s in which it would cause major issues/confusion for them to have joined using a different one?

**Comment by anettlinno** [comment]
Here´s the user case. User receives an invitation email to join public group discussion. She don´t know what the group is all about and what kind of discussions are being held in there. She would like to see what is the topic under discussion and only then decide if she wants to join the discussion or not. 

Current User journey: 
1. User receives an email invitation to join the topic. Email contains the note _You must log in using the same email address ([email address) to access this group_
2. After clicking on "Join group", User is asked to login. When closing the login window from X, black screen is being displayed 

<img width="947" alt="login" src="https://github.com/citizenos/citizenos-fe/assets/45175679/8f765b8d-e7e8-42ca-b517-f06f5770a239">

I agree with @ssin1901 and @BeccaMelhuish that "Join group" button should bring User directly in public group and not ask to go through the login process: 
<img width="936" alt="Public group" src="https://github.com/citizenos/citizenos-fe/assets/45175679/a5ffea06-c064-4e40-be7c-0bfe51c68250">

IF User wants to join discussion, she can decide if she wants to: 
1. Join the group (User is nicely being asked to login)
2. Join the discussion (User is nicely being asked to login)
3. Vote (either use eID or login)

Let´s fix the process and direct User to Public group after clicking "Join group" button. @kevincrepin I didn´t find Group invite flow from Figma. Do we have these designs somewhere please? :)

**Comment by ssin1901** [comment]
@anettlinno thanks for the user story (very clear now) and @BeccaMelhuish thank you so much for sharing this important issue. 

I am actually reconsidering the whole flow based on my expectations. 

For example I would expect a flow like this
1. I receive an invitation to my email.
2. I click on the topic invitation
3. I go directly to the public group I have been invited to (without logging in/registering)
4. If I want to press _Join Group_ only then am I directed to login (and as Becca mentioned does not matter which email I use as long as I am a registered user)

In fact, this is more or less the exact flow that I have as a user who is _not logged in_. So why do we treat users who are invited unequally and create more barriers for them to access the Public Group?

**Comment by anettlinno** [comment]
@ssin1901 I thought this was exactly how I proposed it but thanks for making it clearer :) 

> I agree with @ssin1901 and @BeccaMelhuish that "Join group" button should bring User directly in public group and not ask to go through the login process

**Comment by ssin1901** [comment]
@anettlinno yeah you pretty much already brought out the whole flow! Seems we are all in agreement.

 I will put it on priority to retest this flow once I see the Group invite flow.

**Comment by ilmartyrk** [comment]
@BeccaMelhuish check if it is still relevant or not.
